torchinductor/codegen/triton.py

```diff
@@ -40,25 +40,36 @@ def _print_IndexingDiv(self, expr):
 texpr = TritonPrinter().doprint
 
 
+def triton_dtype(dtype):
+    triton_type_name = str(dtype).split(".")[-1]
+    if triton_type_name == "bool":
+        triton_type_name = "int1"
+    if triton_type_name in ("float16", "bfloat16"):
+        # float16 math is done in float32 inside the kernel
+        triton_type_name = "float32"
+    return f"tl.{triton_type_name}"
+
+
+def triton_constant(value):
+    if value == float("inf"):
+        return 'float("inf")'
+    elif value == float("-inf"):
+        return 'float("-inf")'
+    return repr(value)
+
+
 class TritonOverrides(OpOverrides):
     """Map element-wise ops to Triton"""
 
     @staticmethod
     def to_dtype(x, dtype: torch.dtype):
-        triton_type_name = str(dtype).split(".")[-1]
-        if triton_type_name == "bool":
+        if dtype == torch.bool:
```

Comment:
btw with the latest triton version this workaround for bool is no longer needed

Reply:
Oh great, let's remove it in a week or two to make sure everyone is updated.